src/pages/index.js

```diff
@@ -15,8 +15,8 @@ const featureRows = [
       link: 'docs/concepts/what-is-kilt',
       description: (
         <>
-          Learn about KILT identity components and why they matter.
-          Read on KILT credentials and what they provide.
+          Learn about KILT identity components and why they matter. Read on KILT
```

Comment:
One line one sentence?

Reply:
how did this happen? Must be the auto formatter or something